pyspeckit/cubes/SpectralCube.py

```diff
@@ -774,8 +774,12 @@ def fit_a_pixel(iixy):
             elif errmap is not None:
                 sp.error = np.ones(sp.data.shape) * errmap[y,x]
             else:
-                if verbose_level > 1 and ii==0:
-                    log.warning("WARNING: using data std() as error.", PyspeckitWarning)
+                if ii==0:
+                    # issue the warning only once (ii==0), but always issue
+                    log.warning("WARNING: using data std() as error.  "
+                                "If signal_cut is set, this can result in "
+                                "some pixels not being fit.",
+                                PyspeckitWarning)
```

Comment:
Also, we should actually be using `warn` in many/most cases in which `log.warning` is being used.  `log.warning` reports a warning in the logger, which generally means to the screen, but it does not interact with the python warning system, afaict